shell/packages/sandstorm-email/email.js

```diff
@@ -19,7 +19,7 @@ const makePool = function (mailConfig) {
   const pool = simplesmtp.createClientPool(
     mailConfig.port,
```

Comment:
I guess this should be parseInt(mailConfig.port);

Or alternately, the setting should just be stored as a `Number` in the db.